packages/example/src/design-system/components/TextInput.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/native';
+import { SpatialNavigationNode } from 'react-tv-space-navigation';
+import { TextInput as RNTextInput } from 'react-native';
+import { useRef } from 'react';
+import { Typography } from './Typography';
+import { Box } from './Box';
+
+/**
+ * It works, but it's not perfect.
+ * If you press the back button on Android to dismiss the keyboard,
+ * focus is in a weird state where we keep listening to remote control arrow movements.
+ * Ideally, we'd like to always remove the native focus when the keyboard is dismissed.
+ */
+export const TextInput = ({ label }: { label: string }) => {
+  const ref = useRef<RNTextInput>(null);
+
+  return (
+    <Box>
+      <Typography>{label}</Typography>
+      <SpatialNavigationNode
+        isFocusable
+        onSelect={() => {
+          ref?.current?.focus();
+        }}
+        onFocus={() => {
+          ref?.current?.focus();
+        }}
+        onBlur={() => {
+          ref?.current?.blur();
+        }}
```

Comment:
No changes required, just a question : Do we have an example of blur behaviour in the example app ?

Reply:
Native blur? Not really 😄  This is quite rare. I think the only case is the TextInput.
I haven't included it yet because it's quite ugly on the home page. I'll create additional pages to showcase this.